### DISCUSSION TITLE
Allowing to disable CleanActivitylogCommand command

### DISCUSSION THREAD

**Discussion by Okipa** [spatie/laravel-activitylog#894] [Ideas] [answered]
Hi Spatie team,

I would love to use your package but I don't need logs to be purged after a given period.

Would you accept a PR that disable the `CleanActivitylogCommand ` command if `false` is configured for `delete_records_older_than_days` ?

**Comment by Gummibeer** [comment, marked as answer]
You don't have to disable it - you are in control when the command runs. Don't schedule it and everything fine.

**Reply by Okipa** [reply]
Okay thanks, had in mind it would be automatically scheduled but I it is clear that it can't be added to console Kernel directly from a package 🙃
All good, sorry for the noise!
